executor/runtime/docker/seccomp/seccomp.go

```diff
@@ -228,9 +228,9 @@ type bintree struct {
 }
 
 var _bintree = &bintree{nil, map[string]*bintree{
-	"default.json":          &bintree{defaultJson, map[string]*bintree{}},
-	"fuse-container.json":   &bintree{fuseContainerJson, map[string]*bintree{}},
-	"nested-container.json": &bintree{nestedContainerJson, map[string]*bintree{}},
+	"default.json":          {defaultJson, map[string]*bintree{}},
```

Comment:
As an aside, I filed https://github.com/Netflix/titus-executor/issues/180, which should make this go away in the future.